easybuild/easyconfigs/o/OpenMPI/OpenMPI-3.1.4-GCC-8.3.0.eb

```diff
@@ -17,7 +17,9 @@ checksums = [
 
 dependencies = [
     ('zlib', '1.2.11'),
-    ('hwloc', '1.11.12')
+    ('hwloc', '1.11.12'),
+    ('UCX', '1.6.1'),
+    ('PMIx', '3.1.4'),
```

Comment:
These changes to OpenMPI seem not related to this PR. Were those intended?

Reply:
No, not intended, fixed with force push, sorry...